Documentation/network/bgp-control-plane-failure-scenarios.rst

```diff
@@ -0,0 +1,62 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _bgp_control_plane_failure_scenarios:
+
+BGP Control Plane Failure Scenarios
```

Comment:
To me this feels like it should be part of the existing "BGP Control Plane Troubleshooting Guide", not a separate document. As a user, I would probably have to look into both to find out if one of them contains a solution to my problem.

What I would suggest is to extend the existing "BGP Control Plane Troubleshooting Guide" and have two sections in it:
 - Configuration Issues (the existing content would go under this)
 - Operational Issues (the content of this PR would go there)

Reply:
The timing of reading is different for those two. "Trouble Shooting Guide" is a document when trouble occurs, but for this kind of failure scenario, it is too late to see it when it occurs. The user should be prepared instead.

If it is a part of "Operational Guide" for example, that's understandable, but for "Trouble Shooting" that's questionable for me.

I don't have a super strong opinion, though. @networkop, what do you think? You 👍 to the above comment, so maybe you agree with Rastislav here?

Reply:
ACK, makes sense to me now as well now, thanks for the explanation

Reply:
Create a new Operation Guide doc and move the contents into there.